src/components/molecules/ArticleWrapper/ArticleWrapper.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import newsIcon from "assets/icons/news.png";
+import { Title } from "components/atoms/Title/Title";
+import { ViewWrapper } from "components/atoms/ViewWrapper/ViewWrapper";
+import { Button } from "components/atoms/Button/Button";
+import { ContentWrapper, StyledImg, Wrapper } from "./ArtcileWrapper.styles";
```

Comment:
w tej kwestii zawsze mam 4 bloki importów

```
1. libki zewnętrzne (react, styled-components, formik...)
2. komponenty, utilsy, funkcje
3. style i komponenty z plików .styles.js
4. assety
```

czyli w tym przypadku wyglądałoby to w ten sposób:
```
import React from "react";

import { Title } from "components/atoms/Title/Title";
import { ViewWrapper } from "components/atoms/ViewWrapper/ViewWrapper";
import { Button } from "components/atoms/Button/Button";

import { ContentWrapper, StyledImg, Wrapper } from "./ArtcileWrapper.styles";

import newsIcon from "assets/icons/news.png";
```

ogólnie jest fajny plugin do eslinta który to wymusza, ale na ten moment wystarczy jak będziesz to ogarniać ręcznie